spec/support/requirements_helper.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+RSpec::Matchers.define :have_issue do |field, key, context = {}|
+  context[:styles] ||= %i[form]
+
+  match do |actual_issues|
```

Comment:
can we drop the word 'actual' here? (not sure I see the value its adding)

Reply:
(v minor)

Reply:
It might help to understand the convention for RSpec matchers: https://relishapp.com/rspec/rspec-expectations/v/2-4/docs/custom-matchers/define-matcher#define-a-matcher-with-default-messages

The typical custom matcher would use 'expected' and 'actual'. I've been a bit inconsistent with it, and I think the 'actual' part is pretty obvious from the 'match do' block, so I've updated as you suggest.

Can you confirm it looks OK?